### DISCUSSION TITLE
Will pinocchio v2.7.1 be available on pypi?

### DISCUSSION THREAD

**Discussion by joao-pm-santos96** [stack-of-tasks/pinocchio#2240] [Q&A] [answered]
As of now, the version available on pypi (https://pypi.org/project/pin/) is 2.7.0. Will 2.7.1 be available?

**Comment by nim65s** [comment, marked as answer]
Yes. I'm late about this, sorry. ref. https://github.com/stack-of-tasks/pinocchio/issues/2231

**Reply by joao-pm-santos96** [reply]
No need to apology, I am thankful for your work!
